### PR TITLE
Show a list of all of a persons publications on the person show page

### LINKED ISSUE #111: Link people to publications on people browse details page
In browse people details page, link to that person's publications.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #111

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,6 +1,7 @@
 GIT
   remote: https://github.com/projectblacklight/blacklight.git
-  revision: 5284e21ddd2785923cd1e3836925e1a250397c31
+  revision: b3d53042bf4cbb4a472435999df5162503bde023
```
Comment (justinlittman) [inline Gemfile.lock]:
```
Fetching https://github.com/projectblacklight/blacklight.git
fatal: Could not parse object 'b3d53042bf4cbb4a472435999df5162503bde023'.
Revision b3d53042bf4cbb4a472435999df5162503bde023 does not exist in the
repository https://github.com/projectblacklight/blacklight.git. Maybe you
misspelled it?
```

**Comment by justinlittman** [comment]
I'd suggest that the publications should include publication date and be sorted in descending publication date order.
screen shot 2018-11-08 at 2 40 04 pm

cc @peetucket 

If going to do this, OK if part of this ticket or a separate ticket.

**Comment by jcoyne** [comment]
@justinlittman if you want to expand the scope after the work is done, it makes sense to make a new ticket.

**Comment by peetucket** [comment]
Agreed publication date seems useful, I've created a new ticket: #188

**Pull request merged**